exercise-5/src/com/nayema/Main.java

```diff
@@ -0,0 +1,25 @@
+package com.nayema;
+
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Which Fibonacci number do you want?");
+        int number = scanner.nextInt();
+
+        int num = 0;
```

Comment:
I'd keep the variable naming similar and call the first one `num1` to follow the symmetry of the second `num2` variable.